libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -373,6 +373,8 @@ - (NSError*) prepareErrorForDelegate:(NSData *)data response:(NSURLResponse *)re
 
     // Parse error from data if any.
     if (data) {
+        NSString* newStr = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
+        NSLog(@"%@", newStr);
```

Comment:
Is this debug code?

Reply:
Yessir - I'll be removing all this before marking this ready for review.